src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -601,14 +601,19 @@ public void TestMaxWorkingSet()
             }
         }
 
+	// XXX No longer skip: OSX, FreeBSD
         [Fact]
-        [SkipOnPlatform(TestPlatforms.OSX | TestPlatforms.FreeBSD | TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS, "Getting MaxWorkingSet is not supported on OSX, BSD, iOS, MacCatalyst, and tvOS.")]
+        [SkipOnPlatform(TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS, "Getting MaxWorkingSet is not supported on OSX, BSD, iOS, MacCatalyst, and tvOS.")]
         public void MaxWorkingSet_GetNotStarted_ThrowsInvalidOperationException()
         {
             var process = new Process();
             Assert.Throws<InvalidOperationException>(() => process.MaxWorkingSet);
         }
 
+	// XXX This appears to be testing for incorrect behavoir which has been fixed.
+	// XXX See https://github.com/dotnet/runtime/issues/105422
+	// The correct exception for a new process is: InvalidOperationException
+#if 0
```

Comment:
Is this `#if 0` not allowed?  I don't mean for it to stay long term, just until testing is complete.

Reply:
We wouldn't merge this, if that's what you mean. If you want to experiment with it in CI, that's fine. I'm going to move the PR to be draft until it's ready.

Reply:
OK, though I was hoping the CI stuff could help me with some of this, and now it seems that doesn't happen.

Reply:
CI still runs on drafts

Reply:
Ah, OK. I guess I misunderstood what I saw on the web page.  Thx.